src/js/index.tsx

```diff
@@ -15,37 +26,98 @@ import '../styles.css';
 import TabbedDashboard from './components/TabbedDashboard';
 import SiteHeader from './components/SiteHeader';
 import SiteFooter from './components/SiteFooter';
+import SitePageLoading from './components/SitePageLoading';
 
 import { store } from './store';
+import { useBeaconWithAuthIfAllowed } from '@/hooks';
+
+import { PUBLIC_URL_NO_TRAILING_SLASH, CLIENT_ID, OPENID_CONFIG_URL, AUTH_CALLBACK_URL } from './config';
+
+const SIGN_IN_WINDOW_FEATURES = 'scrollbars=no, toolbar=no, menubar=no, width=800, height=600';
+const CALLBACK_PATH = '/callback';
 
 const LNGS_ARRAY = Object.values(SUPPORTED_LNGS);
 const { Content } = Layout;
 
+const createSessionWorker = () => new Worker(new URL('./workers/tokenRefresh.ts', import.meta.url));
+
 const App = () => {
-  const { lang } = useParams<{ lang?: string }>();
-  const { i18n } = useTranslation();
   const navigate = useNavigate();
 
+  // TRANSLATION
+  const { lang } = useParams<{ lang?: string }>();
+  const { t, i18n } = useTranslation(DEFAULT_TRANSLATION);
+
   useEffect(() => {
+    console.log('lang', lang);
```

Comment:
this is a new console log; was it for debugging, or is it for a more permanent logging purpose (in which case, it should be assigned a level; `console.debug`)